jsonpb.go

```diff
@@ -0,0 +1,198 @@
+// Package gateway contains marshaling code extracted from
+// github.com/grpc-ecosystem/grpc-gateway/runtime and altered
+// to depend on gogo versions of the proto and jsonpb packages.
+package gateway
+
+import (
+	"bytes"
+	"encoding/json"
```

Comment:
Probably in for a following pull request, but do we need the standard encoding/json ?

Reply:
As you imply, I will investigate the warning printouts separately, and they may have something to do with using stdlib json.